docs/parameters.yaml

```
@@ -162,22 +162,23 @@ description: |+
   Cms basically is a file (or asset) discovery service. Each server has a cmsd daemon which
   connect to a master one informing it if a server is available. XRootD asks cms where a file
   could be found and cms works to report back the server for where the file is located.
+  Accepted values: trace, debug, info, warn, error, fatal, panic
 type: string
 default: error
 components: ["origin"]
 ---
 name: Logging.Origin.Scitokens
 description: |+
   Trace level of scitokens debug output within XRootD configuration. This entails token management
-  and security credentials within XRootD.
+  and security credentials within XRootD. Accepted values: trace, debug, info, warn, error, fatal, panic
```

```suggestion
  and security credentials within XRootD. Accepted values: `trace`, `debug`, `info`, `warn`, `error`, `fatal`, `panic`
```

Could you put ` marks between those values to be more readable?